lms/envs/common.py

```diff
@@ -362,6 +362,8 @@
     # feature flag and the advanced setting must both be true for
     # a course to use saved grades.
     'ENABLE_SUBSECTION_GRADES_SAVED': False,
+
+    'RETRY_MAX_ATTEMPTS': 5,
```

Comment:
I don't think this would belong in the `FEATURES` part of the settings—these are usually booleans to tell if a certain feature should be activated or not. Move this down with `RETRY_TIMEOUT`.

Also, let's make these names a little more descriptive:
`RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS`
`RETRY_ACTIVATION_EMAIL_TIMEOUT`